03-account/01-account-management/02-members-management.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Members Management
+description: 'Learn how to manage members and their permissions in your AskUI workspace.'
+---
+
+{/* NOTE: The image references in this document need to be updated with actual screenshots from the workspace settings interface. */}
+
+## Workspace Members Management
+
+In AskUI, workspace members management allows you to control who has access to your workspace and what permissions they have. Members can collaborate on projects, share access to resources, and maintain organized workflows.
+
+## Accessing Members Management
+
+To manage your workspace members:
+
+1. Navigate to the **Settings** tab in the left sidebar.
+2. Locate the **Members** section.
+
+Here you can view all members, their roles, and manage their permissions.
+
+![Screenshot of the Settings tab with Members section open](./images/member-management-settings.png)
+
+## Member Roles and Permissions
+
+Different roles can be assigned to workspace members when you invite them, each with specific permissions:
+
+- **Owner**:
+  - Has complete control over the workspace
+  - Can add and remove other Owners
+  - Can manage subscription and billing
+  - Can delete the workspace
+  - Has all Admin and Member permissions
+
+- **Admin**:
+  - Can add and remove non-Owner members
+  - Can manage workspace resources and settings
+  - Cannot manage subscription or delete workspace
+  - Has all Member permissions
+
+- **Member**:
+  - Can view all workspace resources
+  - Can use the platform functionality
+  - Cannot add or remove other members
+  - Cannot manage workspace settings
+
+Note: Once a member's role is assigned, it cannot be changed. To change a role, you would need to remove the member and invite them again with the desired role.
```

Comment:
What? 

You can not change a member to admin and vice verca? If this is the case. Then it's not acceptable